dm/openapi/spec/dm.yaml

```diff
@@ -1224,6 +1224,31 @@ components:
         - "synced"
         - "binlog_type"
         - "seconds_behind_master"
+    DumpStatus:
+      type: object
+      description: "status of dump unit"
+      properties:
+        total_tables:
+          type: integer
+          format: int64
+        completed_tables:
+          type: number
+          format: double
+        finished_bytes:
+          type: number
+          format: double
```

Comment:
just a question why finished_bytes and completed_tables in double format ?, seems they are integer type on physical sense 🤔

Reply:
because these four fields are float64 at https://github.com/pingcap/tiflow/blob/4ca960d15d8a404b5ab42847e6b4144db851ae20/dm/dm/pb/dmworker.pb.go#L574
I also have a question: what makes the type of these fields are float64